package.json

```
@@ -58,7 +58,7 @@
   },
   "peerDependencies": {
     "express-session": "^1.17.1",
-    "mongodb": "^5.1.0"
+    "mongodb": ">= 5.0.0 < 7"
```

I made a mistake here
```suggestion
    "mongodb": ">= 5.1.0 < 7"
```